framework/SupervisedLearning.py

```diff
@@ -2210,6 +2210,9 @@ def __init__(self,messageHandler,**kwargs):
     """
     supervisedLearning.__init__(self,messageHandler,**kwargs)
     name  = self.initOptionDict.pop('name','')
+    if 'pivotParameter' in self.initOptionDict:
+      # remove pivot parameter if present
+      self.initOptionDict.pop("pivotParameter")
```

Comment:
I did not get here. You changed the way to retrieve the pivotParameter in the ```LearningGate```, but here you start to remove it. why?

Reply:
because the pivotParameter is needed internally for all the TimeDep ROMs, but not for the SKLearn ROMs., since we pass directly the dictionary into the SKL constructors (that complains in case a variable is not recognized (e.g. pivotParameter)

Reply:
Got it.